projects/kit/src/lib/masks/number/processors/create-fullwidth-to-halfwidth-preprocessor.ts

```diff
@@ -0,0 +1,24 @@
+import {MaskitoPreprocessor} from '@maskito/core';
+
+/**
+ * Convert full width numbers like １, ２ to half width numbers 1, 2
+ *
+ * 全角の１、２ののような数字を、半角の1、2のように直すプリプロセッサ
```

Comment:
could you add e2e tests?

Reply:
Of course

Reply:
Adds the test, please confirm if it is completed as you desired, and if okay resolve conversation :)

Reply:
thank you